apis/python/test/common.py

```diff
@@ -310,3 +312,13 @@ def check_training_input_vectors(index_uri: str, expected_training_sample_size:
         assert training_input_vectors.shape[0] == expected_training_sample_size
         assert training_input_vectors.shape[1] == expected_dimensions
         assert not np.isnan(training_input_vectors).any()
+
+def move_local_index_to_new_location(index_uri):
```

Comment:
(not blocking) I believe you could use a pytest fixture for this: https://docs.pytest.org/en/6.2.x/fixture.html

Reply:
Hmm yeah, I think you could. Though I don't totally see what benefit you get here, as this is a function we need to call rather than a variable we want to make available to all tests?